### PR TITLE
resort files and folders

### PR DESCRIPTION
**Pull request by JustDoom**
# Description

draft for resorting the files and folders better, help/suggestions appreciated 
will plan folders/file sorting in the discord then create folders then add/change files so they work

## Checklist

- [x] My contributions follows the style guidelines of this project
- [x] I have performed a self-review of my own contributions (checked for spelling errors, proper linking, etc)
- [x] I have run linting on my branch before submitting the PR

### CODE REVIEW

**Comment by JustDoom** [comment]
this might turn into a complete redo of the files, changing file locations and the text a bit

**Comment by JustDoom** [comment]
oops, its nearly done but not yet

**Comment by JustDoom** [comment]
this should be fine to merge, Ill fix the conflicts when I get home

**Review by Silverarmor — APPROVED** [review]

File: server/guides/other/SETUP.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+![Create folder](/images/server/guides/other/SETUP/CREATE_FOLDER.png)
+
+Once you've done that, drag the server jar into the folder and rename it `server` so we can access it easily in the next step. Next, we need to create the file to run the server jar, you will need to enable file extensions if you haven't already.
+
+![Enable file extensions](/images/server/guides/other/SETUP/ENABLE_FILE_EXTENSIONS.png)
+
+Create a new text document and name it "run", now open it and type
+
+```sh
```
Comment (Kruemmelspalter) [inline server/guides/other/SETUP.md:31]: maybe use `bat` and not `sh` bc it's for a batch file

File: server/guides/other/SETUP.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+This process is slightly different for a modded server so we will go over it here!
+
+# Setup For Linux
+
+> Warning: You should prefer using your package manager (for example `apt` on Ubuntu and Debian) to install java because it's more organized with the package manager
+
+You can do this using the following command
+
+--> `sudo apt install openjdk-xx-jre`
```
Comment (Kruemmelspalter) [inline server/guides/other/SETUP.md:69]: Maybe add either commands for other distros/ package managers or a note that not every distro uses `apt`

File: server/guides/other/SETUP.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+now launch the server / execute `start.sh` again
+
+--> `./start.sh`
+
+You have your server running. To join, launch Minecraft and join the server at `127.0.0.1` or `localhost`
+
+If you want to start the server and want to be able to exit out of the terminal without the server closing, there are two ways with which you can do this
+
+- You can use an in-built linux utility called [tmux](https://github.com/tmux/tmux/wiki). Use this if the server is intended to be temporarily up.
```
Comment (Kruemmelspalter) [inline server/guides/other/SETUP.md:106]: It's not necessarily "in-built" bc on some (or most idk) you need to install it

File: server/guides/other/SETUP.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+- You can register minecraft as a **system service**. Use this if the server is intended to be up 24/7 and requires automatic restarting in case of host shutdowns/failures.
```
Comment (Kruemmelspalter) [inline server/guides/other/SETUP.md:107]: For systemd, it's actually an `unit` and not a `service`

File: server/guides/other/SETUP.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+
+Start a tmux window
+
+`tmux new-session -t sessionName`
+
+`sessionName` being whatever you want to name the instance, for example `minecraft-session`
+
+This will automatically take you inside the tmux session. Start the server using
+
+`java -Xmx1024M -Xms1024M -jar server-jar nogui`
```
Comment (Kruemmelspalter) [inline server/guides/other/SETUP.md:119]: it's `server.jar` not `server-jar`

File: server/guides/other/SETUP.md [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+    ports:
+      - 25565:25565
+    environment:
+      EULA: "TRUE"
+    volumes:
+      # attach the relative directory 'data' to the container's /data path
+      ./data:/data
+```
+
+#### You Build Docker
```
Comment (Kruemmelspalter) [inline server/guides/other/SETUP.md:196]: Maybe "Build it Yourself"

File: server/guides/other/SETUP.md [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+FROM openjdk
+WORKDIR /data
+
+COPY ["server.jar","eula.txt", "start.sh"] ./
+
+EXPOSE 25565
+
+ENTRYPOINT ["/data/start.sh"]
+```
+
```
Comment (Kruemmelspalter) [inline server/guides/other/SETUP.md:210]: Maybe a reminder to use that in the directory with a server installed and maybe add an option to also copy the `world` folder etc.
For example just copy the whole folder

File: server/guides/software/OPTIMIZATION.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# Introduction
+# OPtimization
```
Comment (Kruemmelspalter) [inline server/guides/software/OPTIMIZATION.md:1]: O**p**timization

File: server/guides/software/PICKING.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
-# Intro
+# Choosing a software
```
Comment (Kruemmelspalter) [inline server/guides/software/PICKING.md:1]: Maybe not just "software" but "server software"

**Review by Kruemmelspalter — CHANGES_REQUESTED** [review]

**Review by Kruemmelspalter — APPROVED** [review]

**Pull request merged**